AmazonChimeSDK/AmazonChimeSDK/audiovideo/metric/ObservableMetric.swift

```diff
@@ -57,6 +68,14 @@ import Foundation
             return "videoReceiveBitrate"
         case .videoReceivePacketLossPercent:
             return "videoReceivePacketLossPercent"
+        case .contentShareVideoSendBitrate:
+            return "contentShareVideoSendBitrate"
+        case .contentShareVideoSendPacketLossPercent:
+            return "contentShareVideoSendPacketLossPercent"
+        case .contentShareVideoSendFps:
+            return "contentShareVideoSendFps"
+        case .contentShareVideoSendRttMs:
+            return "contentShareVideoSendRttMs"
         }
```

Comment:
Do you think we should add func isContentShareMetric() to be consistent with Android? I think this will help builders to group the content share metrics if desire.